addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -322,55 +322,55 @@ function clickOnExtraMenuItem(stepOptions, backend = false) {
  * @param {() => TourStep[]} steps The steps of the tour. Has to be a function to avoid direct interpolation of steps. 
```

Comment:
You forgot that one
```suggestion
 * @param {() => TourStep[]} steps The steps of the tour. Has to be a function to avoid direct interpolation of steps.
```

Note that with a good IDE config you can highlight this so it's visually impossible to miss it :+1: 
![image](https://github.com/odoo/odoo/assets/30048408/45a95d0d-9594-4510-ba32-97fb25b7c260)

Reply:
![image](https://github.com/odoo/odoo/assets/127866627/c24ec1db-4311-49c6-bf36-c81350175414)
Note that nearly all the file is uneslinted ... (probably also due that you don't have same configuration as us) ... so it's quite impossible to see it.

Reply:
> so it's quite impossible to see it.

Well, since it was code added in a new pr, it's actually impossible **to not** see it ^^
If you create a new method, you can very well lint it regardless of the rest of the file :grimacing: 

But yeah, it's mainly due to the framework-js custom rules like using only `'` and not `"`.

It's an endless debate, we don't agree with everything that was decided:
- Forcing `'` everywhere "because we can't do better with prettier" https://discord.com/channels/678381219515465750/687338039717920792/1051818992496562266
  It's sad, because using `'` for technical terms and `"` for "display" terms just make it so you can figure what is what in a blink of an eye when reading a codeline. Well, it's a bit the purpose of a guidelines.. You got used to it, then directly figure stuff. A bit like variable name convention (`$variable` for jQuery elements, `variableEl` for dom element etc, it just help understanding instinctively)
- Automatic linter fix on save :grimacing: 
- Some weird rule that enforce usage of `&` condition at the end of the multi line condition instead of the start, while it's really making it harder to read the condition, see https://pep8.org/#break-before-or-after-binary-operator

That's why `website` is not whitelisted in the framework linter tooling thing, we prefer to stick to basic eslint, which is easy to not miss if configured correctly.
Of course, there is some historical linter error still there, but arguably we could clean those once and for all, exactly as we would have to do in a separate commit should we use an automatic linter anyway (to lint everything correctly once before using it).